aptos-move/move-examples/staking/sources/oracle.move

```diff
@@ -0,0 +1,66 @@
+/// Module used to read the price of APT from the oracle.
+module staking::oracle {
+    use aptos_std::math128;
+    use pyth::i64;
+    use pyth::price;
+    use pyth::price_identifier;
+    use pyth::pyth;
+
+    const PRECISION: u128 = 100000000; // 1e8
```

Comment:
why this one has to be 1e8, I guess it is the max decimals pyth can have, probably 2?

Reply:
Pyth prices on Aptos typically have 8 decimals. I triple-checked this for APT at least

Reply:
Yeah, if pyth has a precision of 8, that's good.